cmd/snap-start-preseed/export_test.go

```diff
@@ -0,0 +1,48 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
```

Comment:
let's just call it  snap-preseed, we invoke snapd and stay around anyway, so it's not just starting...